memcache/memcache_test.go

```diff
@@ -308,42 +308,55 @@ func BenchmarkScanGetResponseLine(b *testing.B) {
 func BenchmarkParseGetResponse(b *testing.B) {
```

Comment:
May you show the benchmark comparison before / after, please?

Reply:
> May you show the benchmark comparison before / after, please?

Old pooling vs new pooling with `[]byte`:
```
$ benchstat old.txt new.txt 
name                 old time/op    new time/op    delta
ParseGetResponse-16     251ns ± 0%     276ns ± 0%   ~     (p=1.000 n=1+1)

name                 old alloc/op   new alloc/op   delta
ParseGetResponse-16     88.0B ± 0%    112.0B ± 0%   ~     (p=1.000 n=1+1)

name                 old allocs/op  new allocs/op  delta
ParseGetResponse-16      2.00 ± 0%      3.00 ± 0%   ~     (p=1.000 n=1+1)
```

Old pooling vs new pooling with `*[]byte`
```
$ benchstat old.txt new.txt 
name                 old time/op    new time/op    delta
ParseGetResponse-16     251ns ± 0%     255ns ± 0%   ~     (p=1.000 n=1+1)

name                 old alloc/op   new alloc/op   delta
ParseGetResponse-16     88.0B ± 0%     88.0B ± 0%   ~     (all equal)

name                 old allocs/op  new allocs/op  delta
ParseGetResponse-16      2.00 ± 0%      2.00 ± 0%   ~     (all equal)
```

Reply:
The delta reporting is misleading. There's actually a difference.